README.md

```diff
@@ -2,15 +2,12 @@
 
 Code repo for Delaware River Basin information theory code and machine learning models that predict estuary salinity
 
-This information is preliminary or provisional and is subject to revision. It is being provided to meet the need for timely best science. The information has not received final approval by the U.S. Geological Survey (USGS) and is provided on the condition that neither the USGS nor the U.S. Government shall be held liable for any damages resulting from the authorized or unauthorized use of the information.
-
 The steps for reproducing the results from the manuscript are detailed below. To retrain a new model use all steps. To analyze results from the manuscript run steps 1-3, then use steps 9-10 for analysis. Model ouptut used for the manuscript is located in `03_model/out/Manuscript_Results` 
 
 #### Steps for reproducibility:
 
 1) clone this repo using `git clone git@github.com:USGS-R/drb-estuary-salinity-ml.git --recurse-submodules`, the `--recurse-submodules` command initiates and updates the `river-dl` submodule housed in `03_model/src/` directory 
```

Comment:
When we release this model archive, the users will be downloading it from ScienceBase. Do you know if there is any way to recurse submodules if the files are not coming from Github? I actually think we don't want to update the `river-dl` submodule because we want a frozen snapshot of the way it existed when you made this model archive - so really we just want to initiate it somehow.

Reply:
Ok I'll look into this, maybe I can include the hash of the last commit and I can include instructions to load it from there